draft-erdtman-jose-cleartext-jws.xml

```diff
@@ -191,8 +230,8 @@
             algorithm used to construct the JWS Signature.
           </t>
           <t>
-            Add the <spanx style="verb">__cleartext_signature</spanx> member to the signature object with the encoded
-            Cleartext JWS signature, BASE64URL(JSC Signature).
+            Add the <spanx style="verb">signature</spanx> member to the signature object with the encoded
+            Cleartext JWS signature and assign it the string value BASE64URL(signature).
```

Comment:
In JWS it sayas BASE64URL(JWS Signature) not sure that would be 100% correct so this is okay for me, just wanted to mention it.

Reply:
I can that :-)